echo-rest/src/main/java/com/netflix/spinnaker/echo/events/RestEventListener.java

```diff
@@ -74,21 +67,9 @@ public void processEvent(Event event) {
               try {
                 Map<String, Object> eventMap = mapper.convertValue(event, Map.class);
 
-                if (service.getConfig().getFlatten() && restEventParser.isPresent()) {
-                  RestEventParser.FlatEvent flatEvent =
-                      restEventParser
-                          .get()
-                          .flattenEvent(mapper.convertValue(event, RestEventParser.Event.class));
-                  eventMap.put("content", flatEvent.getContent());
-                  eventMap.put("details", flatEvent.getDetails());
-                } else if (service.getConfig().getFlatten()) {
+                if (service.getConfig().getFlatten()) {
                   eventMap.put("content", mapper.writeValueAsString(eventMap.get("content")));
                   eventMap.put("details", mapper.writeValueAsString(eventMap.get("details")));
-                } else if (restEventParser.isPresent()) {
-                  eventMap =
-                      restEventParser
-                          .get()
-                          .parseEvent(mapper.convertValue(event, RestEventParser.Event.class));
```

Comment:
I see this code being removed here, but I don't recall seeing it put somewhere else. It's concerning to me that this didn't break any tests, either?

Reply:
This is the integration for the first extension point I added to echo (`RestEventParser`), but I think there's more value in pulling it out and redesigning how we build the echo-api.  It's not being used at all right now.